libs/pages/application/src/lib/ui/page-settings-advanced/page-settings-advanced.tsx

```diff
@@ -0,0 +1,186 @@
+import { ApplicationAdvancedSettings } from 'qovery-typescript-axios'
+import { useState } from 'react'
+import { Controller, useFormContext } from 'react-hook-form'
+import { LoadingStatus } from '@console/shared/interfaces'
+import {
+  CopyToClipboard,
+  EditionTable,
+  EditionTableRow,
+  HelpSection,
+  InputTextSmall,
+  InputToggle,
+  LoaderSpinner,
+  StickyActionFormToaster,
+  Tooltip,
+} from '@console/shared/ui'
+
+export interface PageSettingsAdvancedProps {
+  keys?: string[]
+  defaultAdvancedSettings?: ApplicationAdvancedSettings
+  advancedSettings?: ApplicationAdvancedSettings
+  loading: LoadingStatus
+  onSubmit?: () => void
+  discardChanges: () => void
+}
+
+export function PageSettingsAdvanced(props: PageSettingsAdvancedProps) {
+  const { control, formState } = useFormContext()
+  const [showOverriddenOnly, toggleShowOverriddenOnly] = useState(false)
+
+  let table: EditionTableRow[] = [
+    {
+      cells: [
+        {
+          content: 'Settings',
+        },
+        {
+          content: 'Default Value',
+        },
+        {
+          content: <span className="px-3">Value</span>,
+          className: '!p-1',
+        },
+      ],
+      className: 'font-medium',
+    },
+  ]
+
+  const tableBody: EditionTableRow[] = props.keys
```

Comment:
The name of the props should be `EditionTableRowProps` we need Props at the end of the name

Reply:
mmh not sure about this, here `EditionTableRow` is the interface of one of the property of `EditionTableProps` for me it's not a Prop

Reply:
Yes but for me like this it can be confused with an composant, maybe it can be "EditionTableRowInterface"